modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Validation/Validation.es.js

```diff
@@ -28,10 +28,12 @@ const defaultValue = {
 	parameter: {},
 };
 
-const ValidationWithProvider = (props) => (
-	<PageProvider value={{editingLanguageId: 'en_US'}}>
-		<Validation {...props} />
-	</PageProvider>
+const ValidationWithProvider = ({validations, ...props}) => (
+	<FormProvider initialState={{validations}}>
+		<PageProvider value={{editingLanguageId: 'en_US'}}>
+			<Validation {...props} />
+		</PageProvider>
```

Comment:
The page PageProvider is not being used by the `Validation` component, so you can just remove it
```suggestion
		<Validation {...props} />
```

Reply:
Done. Thanks for your suggestion, @miltonmc